examples/openshift-data-foundation/deployment/README.md

```diff
@@ -0,0 +1,130 @@
+# Deploying and Managing Openshift Data Foundation (Test)
+
+This example shows how to Deploy and Manage your Openshift Data Foundation add-on on a Red Hat Openshift Cluster. Note this template is still in development, so please be advised before using in production.
+
+This sample configuration will deploy the add-on, scale and upgrade it using the "ibm_container_addons" and kubernetes_manifest from the ibm terraform provider and kubernetes provider respectively.
+
+## Usage
+
+To run this example you need to execute:
+
+```bash
+$ terraform init
+$ terraform plan --var-file input.tfvars
+$ terraform apply --var-file input.tfvars
+```
+
+Run `terraform destroy` when you don't need these resources.
+
+## Example usage
+
+The following use cases are handled by editing the input.tfvars file. Terraform resources need to be touched.
+
+### Create & Deployment:
+
+The default input.tfvars is given below, the user should just change the value of the parameters in accorandance to their requirment. 
+
+```hcl
+# For add-on deployment
+odfVersion = "4.10.0"
+
+# For CRD Creation and Management
+kube_config_path = "~/.kube/config"
+autoDiscoverDevices = "false"
+billingType = "advanced"
+clusterEncryption = "false"
+hpcsBaseUrl = null
+hpcsEncryption = "false"
+hpcsInstanceId = null
+hpcsSecretName = null
+hpcsServiceName = null
+hpcsTokenUrl = null
+ignoreNoobaa = "false"
+numOfOsd = "1"
+ocsUpgrade = "false"
+osdDevicePaths = null
+osdSize = "100Gi"
+osdStorageClassName = "ibmc-vpc-block-metro-10iops-tier"
+workerNodes = null
+```
+### Editing the Ocscluster custom resource
+
+The following variables in the `input.tfvars` file can be edited
+
+* numOfOsd - To scale your storage
+* workerNodes - To increase the number of workerNodes with ODF
+
+```hcl
+# For CRD Management
+numOfOsd = "1" -> "2"
+workerNodes = null -> "worker_1_ID,worker_2_ID"
```

Comment:
change the example to 
```
workerNodes = "worker_1_ID,worker_2_ID,worker_1_ID,worker_3_ID" -> "worker_1_ID,worker_2_ID,worker_1_ID,worker_3_ID,worker_4_ID,worker_5_ID,worker_1_ID,worker_6_ID"
```
Also add a note for workerNodes that it should be updated only if workerNodes was provided as input during ODF deploy.